scrapers.js

```diff
@@ -40,6 +40,27 @@ const scrapers = [
       return counties;
     }
   },
+  {
+    state: 'IL',
+    country: 'USA',
+    url: 'http://www.dph.illinois.gov/sites/default/files/COVID19/COVID19CountyResults.json',
+    async scraper() {
+      const data = await fetch.json(this.url);
+
+      const counties = [];
+      // do stuff
+      for (const county of data.characteristics_by_county.values) {
+        counties.push({
+          county: `${county.County} County`,
```

Comment:
Please use `transform.addCounty()` in case they decided they want to add `County` to the field later (avoids `Whatever County County`
```suggestion
          county: transform.addCounty(county.County),
```